cmd/cstor-pool-mgmt/pool/v1alpha2/status.go

```diff
@@ -0,0 +1,105 @@
+package v1alpha2
+
+import (
+	"strings"
+
+	api "github.com/openebs/maya/pkg/apis/openebs.io/v1alpha2"
+	zfs "github.com/openebs/maya/pkg/zfs/cmd/v1alpha1"
+)
+
+const (
+	// StatusNoPoolsAvailable .. not pools available
+	StatusNoPoolsAvailable = "no pools available"
+	// PoolStatusDegraded .. pool is degraded
+	PoolStatusDegraded = "DEGRADED"
+	// PoolStatusFaulted .. pool is in faulty state
+	PoolStatusFaulted = "FAULTED"
+	// PoolStatusOffline .. pool is in offline state
+	PoolStatusOffline = "OFFLINE"
+	// PoolStatusOnline .. pool is online
+	PoolStatusOnline = "ONLINE"
+	// PoolStatusRemoved .. pool is in removed state
+	PoolStatusRemoved = "REMOVED"
+	// PoolStatusUnavail .. pool is unavailable
+	PoolStatusUnavail = "UNAVAIL"
+)
+
+// GetStatus return status of the pool
+func GetStatus(csp *api.CStorNPool) (api.CStorPoolStatus, error) {
+	var status api.CStorPoolStatus
+
+	ret, err := zfs.NewPoolStatus().
+		WithPool(PoolName(csp)).
+		Execute()
+	if err != nil {
+		return status, err
+	}
+
+	status.Phase = parsePoolStatus(string(ret))
+
+	freeSize, er := getPropertyValue(csp, "free")
```

Comment:
getPropertyValue sounds like getting property from csp.. but, that doesn't seems to be the case here..
passing just name should be good

Reply:
I mean - getPropertyValue taking csp as input sounds like getting property from it

Reply:
ok. I will change argument to pool name only.